.github/workflows/build-sw-container.yml

```diff
@@ -16,53 +16,46 @@ jobs:
     - name: make date tag
       id: mkdatetag
       run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
-
+    
   build:
     runs-on: ubuntu-latest
-    needs: [make-date-tag]
-    if: startsWith(github.repository, 'opensciencegrid/')
+    continue-on-error: true
```

Comment:
If the build fails, I'm not keen on moving onto the pushes. If it's a transient failure, we can always rerun the CI